webapp/commonComponents/form/languageDropdown.js

```diff
@@ -6,13 +6,13 @@ import { getLanguageLabel, languages } from '../../../common/app/languages'
 
 const LanguageDropdown = (props) => {
 
-  const {selection, onChange, validation} = props
+  const { selection, onChange, validation, placeholder } = props
 
   const selectedItem = selection
-    ? {key: selection, value: getLanguageLabel(selection)}
+    ? { key: selection, value: getLanguageLabel(selection) }
     : null
 
-  return <Dropdown placeholder="Language"
+  return <Dropdown placeholder={placeholder}
```

Comment:
since the placeholder of LanguageDropdown won't change, I wouldn't pass the placeholder from outside, I'd rather used i18n here to take the placeholder label